templates/medical-bill/templates/main.tpl

```diff
@@ -0,0 +1,282 @@
+
+{{template "simple-paragraph" dict "Margin" "5 0 5 0" "LineHeight" 1.0 "Font" "helvetica-bold" "FontSize" 14 "TextColor" "#000000" "Text" "How to Apply"}}
+{{template "simple-paragraph" dict "Margin" "0 0 10 0" "LineHeight" 1.0 "Font" "helvetica" "FontSize" 12 "TextColor" "#000000" "Text" "Financial Assistance applications may be obtained/completed/submitted as follows:"}}
+
+<list indent="5">
+    <list-marker font-size="12">• </list-marker>
+    <list-item>
+        <paragraph>
+            <text-chunk font-size="12">Obtain an application at The Sample Medical Center's Patient Financial Services Department located at Main Street 123</text-chunk>
+        </paragraph>
+    </list-item>
+    <list-item>
+        <paragraph>
+            <text-chunk font-size="12">Request to have an application by mail at: 123 Main Street, Anywhere, NY 12345 - 6789.</text-chunk>
+        </paragraph>
+    </list-item>
+    <list-item>
+        <paragraph>
+            <text-chunk font-size="12">Request to have an application mailed to you by calling 123 - 456 - 7890. Our hours of operation are: Monday-Friday, 8:30a.m.-4:30p.m.</text-chunk>
+        </paragraph>
+    </list-item>
+    <list-item>
+        <paragraph>
+            <text-chunk font-size="12">Download an application through the Sample Medical Center's website:
+</text-chunk>
+            <text-chunk font-size="12" link="url('https://www.ourwebsite.com/PatientFinancialServices.aspx')">https://www.ourwebsite.com/PatientFinancialServices.aspx</text-chunk>
+        </paragraph>
+    </list-item>
+</list>
+
+<paragraph>
+    <text-chunk font-size="12">Patient Financial Service Counselors are available Monday through Friday, 8:30 a.m. to 4:30 pm via telephone (123) 456-7890 to address questions related to the Financial Assistance Program.
+Please feel free to email us at:</text-chunk>
+    <text-chunk font-size="12" link="url('mailto:businessoffice@ourwebsite.com')">businessoffice@ourwebsite.com.</text-chunk>
+</paragraph>
+
+{{template "simple-paragraph" dict "Margin" "15 0 5 0" "LineHeight" 1.0 "Font" "helvetica-bold" "FontSize" 14 "TextColor" "#000000" "Text" "Section 1557 — Notice of Nondiscrimination"}}
+{{template "simple-paragraph" dict "Margin" "0 0 10 0" "LineHeight" 1.0 "Font" "helvetica" "FontSize" 12 "TextColor" "#000000" "Text" "The Sample Medical Center complies with applicable Federal civil right laws and does not discriminate on the basis of race, color, national origin, age, disability, or sex."}}
+
+
+{{template "simple-paragraph" dict "Margin" "15 0 0 0" "LineHeight" 1.0 "Font" "helvetica" "FontSize" 10 "TextColor" "#000000" "Text" "If any of this following has changed since your last statement, please indicate..."}}
+
+<table columns="2">
```

Comment:
Some margin may be needed here. 
The text `If any of this following has...` is half covered by the table on the generated pdf.